draft-iab-protocol-maintenance.md

```diff
@@ -80,12 +80,12 @@ design of interoperable protocols in general.
 Time and experience shows that negative consequences to interoperability
 accumulate over time if implementations apply the robustness principle.  This
 problem originates from an assumption implicit in the principle that it is not
-possible to affect change in a system the size of the Internet.  That is, the
-idea that once a protocol specification is published, changes that might require
-existing implementations to change are not feasible.
+possible to effect change in a system the size of the Internet.  It assumes
```

Comment:
Very minor nit: while effect is also correct, I find affect to be clearer. Happy to be overruled by native speakers (in which case I'll learn something :-) ) and happy to have this merged with either effect or affect

```suggestion
possible to affect change in a system the size of the Internet.  It assumes
```

Reply:
This is definitely "effect change", IMO =)

https://writingexplained.org/affect-change-or-effect-change

> What does affect change mean? Affect change is an incorrect version of the phrase "effect change".

Reply:
Ah I see. `affect a system` means roughly the same as `effect change in a system` whereas `affect change` is redundant. Thank you :) Marking as resolved.